samples/msal-node-samples/auth-code-distributed-cache/src/PartitionManager.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccountEntity, AccountInfo } from "@azure/msal-common";
+import { IPartitionManager } from "@azure/msal-node";
+
+import RedisClientWrapper from "./RedisClientWrapper";
+
+const SESSION_KEY_PREFIX = "sess:";
+const EMPTY_STRING = "";
```

Comment:
Could you import this from common?

Reply:
Or don't use a const

Reply:
Everyone always tells me to use const for empty string!

Reply:
It's supposed to be a visual aid when skimming code, among other things, but it's one of those things where every side is right I think 😄  

Can't/won't import directly from common library in a sample, so will leave this as is.